share/jupyterhub/templates/spawn_pending.html

```diff
@@ -17,7 +17,12 @@
             <span class="sr-only"><span id="sr-progress">0%</span> Complete</span>
           </div>
         </div>
-        <p id="progress-message"></p>
+        <p id="progress-message">Server requested</p>
+        <!-- Stop Server Button -->
+        <div class="mt-3">
```

Comment:
Maybe move the stop button either _above_ progress or below events? As it is now, it is in the middle of the progress, between the latest event and the rest of the events.

Reply:
Thanks for the change, This is neither above the progress bar nor below the events, it is still in the middle of progress information.